pkg/kubectl/describe.go

```diff
@@ -830,7 +830,7 @@ func (d *PersistentVolumeDescriber) Describe(namespace, name string, describerSe
 		w := &PrefixWriter{out}
 		w.Write(LEVEL_0, "Name:\t%s\n", pv.Name)
 		printLabelsMultiline(w, "Labels", pv.Labels)
-		w.Write(LEVEL_0, "StorageClass:\t%s\n", storageutil.GetStorageClassAnnotation(pv.ObjectMeta))
```

Comment:
Ok I think I see in conversion.go, it will convert beta annotation to spec.storageclass.  So newer kubectl reading from old API will work.

And old kubectl sending beta annotation -> new API will work.  But what about old kubectl reading from new API that doesn't use beta annotation?